code/src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Header = () => {
+
+    return (
+        <div>
+
+            <h1>
+            New albums & singles
+            </h1>
+            <hr />
```

Comment:
this is a good one, didn't thought about this tag. Instead used a border bottom in css, which in the end caused some styling problems